apps/api/entrypoint.sh

```
@@ -35,6 +35,12 @@ if [ -n "$API_DB_PASSWORD" ]; then
   export DATABASE_URL="postgresql+psycopg://clevis_api:$(_urlenc "$API_DB_PASSWORD")@db:5432/${_db_name_enc}"
 fi
 
+# Issue #191/S3: webhook ingestion queue. Built here (not left as a directly-set
+# REDIS_URL) so the raw password only ever needs to be set once, the same shape as
+# DB_USER/DB_PASSWORD/DB_NAME above -- REDIS_URL itself stays a "local dev outside
+# Docker only" var (see .env.example), same as DATABASE_URL.
+export REDIS_URL="redis://:$(_urlenc "$REDIS_PASSWORD")@redis:6379/0"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)